test/path_filtering.go

```diff
@@ -90,11 +92,17 @@ func getFeatureGate(namespace string) (string, error) {
 // stablePathFilter returns true for any example that should be allowed to run
 // when "enable-api-fields" is "stable".
 func stablePathFilter(p string) bool {
-	return !(strings.Contains(p, "/alpha/") || strings.Contains(p, "/beta/"))
+	return !strings.Contains(p, "/alpha/") && !strings.Contains(p, "/beta/")
 }
 
 // alphaPathFilter returns true for any example that should be allowed to run
 // when "enable-api-fields" is "alpha".
 func alphaPathFilter(p string) bool {
 	return strings.Contains(p, "/alpha/") || strings.Contains(p, "/beta/") || stablePathFilter(p)
 }
+
+// betaPathFilter returns true for any example that should be allowed to run
+// when "enable-api-fields" is "beta".
+func betaPathFilter(p string) bool {
+	return strings.Contains(p, "/beta/") || stablePathFilter(p)
```

Comment:
I think this can be simplified to `!strings.Contains(p, "/alpha/")`

Reply:
Yes! Thanks